<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugHardware</class>
 <widget class="QWidget" name="DebugHardware">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_22">
       <property name="styleSheet">
        <string notr="true">QLabel {color:white}</string>
       </property>
       <property name="text">
        <string>CPU实时电压</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_CPU_VCC">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit,QDateTimeEdit{
border:0px solid #242424;
border-radius:3px;
padding:2px;
color:#FFFFFF;
selection-background-color:#1C1C28;
selection-color:#0EC3FD;
}</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_24">
       <property name="styleSheet">
        <string notr="true">QLabel {color:white}</string>
       </property>
       <property name="text">
        <string notr="true">mV</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_23">
       <property name="styleSheet">
        <string notr="true">QLabel {color:white}</string>
       </property>
       <property name="text">
        <string>CPU实时温度</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_CPU_TEMP">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit,QDateTimeEdit{
border:0px solid #242424;
border-radius:3px;
padding:2px;
color:#FFFFFF;
selection-background-color:#1C1C28;
selection-color:#0EC3FD;
}</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_25">
       <property name="styleSheet">
        <string notr="true">QLabel {color:white}</string>
       </property>
       <property name="text">
        <string notr="true">℃</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_26">
       <property name="styleSheet">
        <string notr="true">QLabel {color:white}</string>
       </property>
       <property name="text">
        <string>电池实时电压</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_BATT_VCC">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit,QDateTimeEdit{
border:0px solid #242424;
border-radius:3px;
padding:2px;
color:#FFFFFF;
selection-background-color:#1C1C28;
selection-color:#0EC3FD;
}</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_27">
       <property name="styleSheet">
        <string notr="true">QLabel {color:white}</string>
       </property>
       <property name="text">
        <string notr="true">V</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_28">
       <property name="styleSheet">
        <string notr="true">QLabel {color:white}</string>
       </property>
       <property name="text">
        <string>电池实时电流</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_BATT_CUR">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit,QDateTimeEdit{
border:0px solid #242424;
border-radius:3px;
padding:2px;
color:#FFFFFF;
selection-background-color:#1C1C28;
selection-color:#0EC3FD;
}</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_29">
       <property name="styleSheet">
        <string notr="true">QLabel {color:white}</string>
       </property>
       <property name="text">
        <string notr="true">mA</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_30">
       <property name="styleSheet">
        <string notr="true">QLabel {color:white}</string>
       </property>
       <property name="text">
        <string>电池实时功率</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_BATT_P">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit,QDateTimeEdit{
border:0px solid #242424;
border-radius:3px;
padding:2px;
color:#FFFFFF;
selection-background-color:#1C1C28;
selection-color:#0EC3FD;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_33">
       <property name="styleSheet">
        <string notr="true">QLabel {color:white}</string>
       </property>
       <property name="text">
        <string notr="true">W</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_19">
       <property name="styleSheet">
        <string notr="true">QLabel {color:white}</string>
       </property>
       <property name="text">
        <string>编 译 时 间</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEdit_CompileTime">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit,QDateTimeEdit{
border:0px solid #242424;
border-radius:3px;
padding:2px;
color:#FFFFFF;
selection-background-color:#1C1C28;
selection-color:#0EC3FD;
}</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>86</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>7</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
